c/src/lib/libbsp/arm/shared/stm32fxxxx/include/hal-fatal-error-handler.h

```diff
@@ -0,0 +1,28 @@
+/**
+ * @file hal-fatal-error-handler.h
+ *
+ * @ingroup error
+ *
+ * @brief Error handler for CPU exceptions
+ *
+ */
+
+/*
+ * Copyright (c) 2015 Vecna Technologies, Inc.
+ *
+ * Author: Sudarshan Rajagopalan
+ *
+ * The license and distribution terms for this file may be
+ * found in the file LICENSE in this distribution or at
+ * http://www.rtems.com/license/LICENSE.
+ */
+
+#ifndef HAL_FATAL_ERROR_HANDLER_H_
+#define HAL_FATAL_ERROR_HANDLER_H_
+
+/**
+ * @brief HAL fatal error handler function.
+ */
+rtems_status_code stm32f_initialize_user_extensions( void );
```

Comment:
The name of the implementation of this function is stm32_initialize_extensions in the .c file.  Please fix one of them so they have a consistent name.